experimental/uefi/kernel/src/lib.rs

```diff
@@ -60,10 +65,23 @@ pub fn start_kernel<E: boot::E820Entry, B: boot::BootInfo<E>>(info: &B) -> ! {
 
 fn main<E: boot::E820Entry, B: boot::BootInfo<E>>(info: &B) -> ! {
     info!("In main! Boot protocol:  {}", info.protocol());
-    let serial = serial::Serial::new();
     let attestation_behavior =
         AttestationBehavior::create(EmptyAttestationGenerator, EmptyAttestationVerifier);
-    runtime::framing::handle_frames(serial, attestation_behavior).unwrap();
+    runtime::framing::handle_frames(get_channel(), attestation_behavior).unwrap();
+}
+
+#[cfg(not(feature = "vsock_channel"))]
```

Comment:
Hu, took me some time to see the difference between the two `get_channel`, i.e., the feature gating with not. Maybe some brief comment with when/why we fall back on that one would be nice.

Reply:
Added some comments.